gradoop-flink/src/main/java/org/gradoop/flink/model/impl/operators/grouping/Grouping.java

```diff
@@ -659,26 +660,29 @@ public GroupingBuilder addEdgeAggregateFunction(AggregateFunction aggregateFunct
       V extends Vertex,
       E extends Edge,
       LG extends BaseGraph<G, V, E, LG, GC>,
-      GC extends BaseGraphCollection<G, V, E, LG, GC>> Grouping<G, V, E, LG, GC> build() {
+      GC extends BaseGraphCollection<G, V, E, LG, GC>> UnaryBaseGraphToBaseGraphOperator<LG> build() {
```

Comment:
@galpha What is your opinion here? I think since we separated the old grouping and the keyed grouping, there is no need to access the keyed grouping via the old one. It's only a convenience feature and changes the API (more specific the return type of the grouping).

My vote is for remove all changes from classes `Grouping` and `GroupingStrategy`.